README.md

```diff
@@ -2,68 +2,101 @@
 
 TIMDEX Pipeline Lambdas is a collection of lambdas used in the TIMDEX Ingest Pipeline.
 
-## Format Handler
+## Format Input Handler
 
-Takes an input from EventBridge and formats the event data to work throughout the entire ingest pipeline.
+Takes input JSON (usually from EventBridge although it can be passed to a manual Step Function execution), and returns reformatted JSON matching the expected input data needed for the remaining steps in the TIMDEX pipeline Step Function.
 
-### Example Format input
+### Input fields
+
+#### Required fields
+
+- `harvest-type`: Must be one of `["full", "daily"]`. The provided harvest type is used in the input/output file naming scheme for all steps of the pipeline. It also determines the harvest logic as follows:
+  - `full`: Will perform a full harvest of all records from the provided `oai-pmh-host`.
+  - `daily`: Will harvest only records added to or updated in the provided `oai-pmh-host` during the previous calendar day. Previous day is relative to the provided `time` field date, *not* the date this process is run, although those will be equivalent in most cases.
+- `opensearch-url`: The full URL for the OpenSearch instance to ingest records into during the load step.
+- `output-bucket`: The name of the S3 bucket (just the name, not the URI) in which to retrieve and deposit input/output files.
+- `starting-step`: Must be one of `["harvest", "transform", "load"]`:
+  - `harvest`: Will start the pipeline at the harvest step and continue through transform and load.
+  - `transform`: Will skip harvest and start the pipeline at the transform step, using existing harvest output file matching the provided source, harvest type, and time, and continue through load.
+  - `load`: Will skip harvest and transform steps and start the pipeline at the load step, using existing transform output file matching the provided source, harvest type, and time.
+- `source`: Short name for the source repository, must match one of the source names configured for use in transform and load apps. The provided source is passed to the transform and load app CLI commands, and is also used in the input/output file naming scheme for all steps of the pipeline.
+  - *Note*: if provided source is "aspace", a method option is passed to the harvest command (if starting at the harvest step) to ensure that we use the slower but more accurate "get" harvest method instead of the default "list" method used for all other sources.
```

Comment:
Likely unnecessary but it may be worth clarifying that the accuracy issue is specific to aspace's quirks and not the method itself since we haven't had accuracy issues with any other source to my knowledge